README.md

```diff
@@ -1,2 +1,172 @@
 # db-muffler
-Tool to obscure values in your SQL dump file
+
+> Tool to obscure values in your SQL dump file
+
+## Idea
+
+This tool will allow to obscure specific fileds within INSERT statements of your SQL dump file.
+Could be a good tool to hide sensitive data or just replace your dev data with more realistic one.
+
+## Installation
+
+Global installation _(suggested)_
+
+```
+$ npm i -g db-muffler
+$ db-muffler
+```
+
+Local installation
+
+```
+$ npm install db-muffler
+$ node_modules/.bin/db-muffler
+```
+
+## Usage
+
+```
+Usage
+    $ db-muffler <file_path>
+
+Options
+    --test, -t    Test run which would not overwrite input file
+    --output, -o  Output file where the result will be store. Default value is the input file name
+    --config, -c  Config file where developer can store rules on how to obscure DB data. Default value is ./db-muffler.json
+
+Examples
+    $ db-muffler ./inputfile.sql
+    Will use ./db-muffler.json config file and then will run muffler to obscure
+    data and then output the result in inputfile.sql
+
+    $ db-muffler ./inputfile.sql --test
+    Will use ./db-muffler.json config file and then will run muffler to obscure
+    data and then output the result in stdout
+
+    $ db-muffler ./inputfile.sql -o ./outputfile.sql
+    Will use ./db-muffler.json config file and then will run muffler to obscure
+    data and then output the result into outputfile.sql
+
+    $ db-muffler ./inputfile.sql -o ./outputfile.sql -c ./config.json
+    Will use config.json config file and then will run muffler to obscure
+    data and then output the result into outputfile.sql
+```
+
+## PostgreSQL cheatsheet to generate SQL dump files
+
+-   To create SQL file for your DB
+
+```
+pg_dump db_name -U my_username -h localhost -F c > my.dump
+pg_restore -f my.sql my.dump
+```
+
+-   To create SQL file for a specific table
+
+```
+pg_dump db_name -U my_username -h localhost -F c --column-inserts -a -t public.table_name > my.dump
+pg_restore -f my.sql my.dump
+```
+
+#### Sample run
+
+```
+node index-cli.js ./example/sample.sql -o ./example/output.sql -c ./example/sampleConfig.json
```

Comment:
Should this be `db-muffler` ?

Reply:
only if it is installed globally. But if you just cloned the repo this command will work. I need to add more documentation explaining on what sample run it is.